man/subuid.5.xml

```diff
@@ -48,11 +53,37 @@
   </refmeta>
   <refnamediv id='name'>
     <refname>subuid</refname>
-    <refpurpose>the subordinate uid file</refpurpose>
+    <refpurpose>the configuration for subordinate user ids</refpurpose>
   </refnamediv>
 
   <refsect1 id='description'>
     <title>DESCRIPTION</title>
+    <para>
+      Subuid links a user id from a parent namespace to a list of user ids from
```

Comment:
'links' doesn't seem like quite the right word.  Also subuid only can authorize passing uids into a child namespace, of course, which might be a bit confusing in this sentence.  How would you feel about something like

Subuid authorizes a user id to map lists of user ids from its namespace into child namespaces.

Other than that, this looks great.  Did you want to also do this for man/subgid.5.xml?

Reply:
> Subuid authorizes a user id to map lists of user ids from its namespace into child namespaces.

This variant reads better, imo.

Reply:
I've updated this part.